magefiles/magefile.go

```diff
@@ -469,3 +469,8 @@ type CloudActions mg.Namespace
 func (CloudActions) Generate() error {
 	return sh.RunWith(ENV, "go", "run", "-tags=mage_cloudactions", "./magefiles")
 }
+
+// VEX generates a VEX document for Trivy
+func VEX(_ context.Context, dir string) error {
+	return sh.RunWith(ENV, "go", "run", "-tags=mage_vex", "./magefiles/vex.go", "--dir", dir)
+}
```

Comment:
what about `output` flag?

Reply:
We probably don't need the output flag because the VEX file should be written under `.vex/`. I added it for local debugging, though.

Reply:
>All arguments are mandatory and must be specified in the order they appear in the function.

All arguments are mandatory in Mage.
https://magefile.org/targets/

It means we have to specify `.vex/vex.json` every time.